config/config.example.toml

```diff
@@ -74,10 +74,11 @@ max_feed_count = 200              # The maximum number of frames that will be fe
 
 # This section provides configs for currency conversion api
 [forex_api]
-call_delay = 21600                # Expiration time for data in cache as well as redis in seconds
 api_key = ""                      # Api key for making request to foreign exchange Api
 fallback_api_key = ""             # Api key for the fallback service
-redis_lock_timeout = 100          # Redis remains write locked for 100 s once the acquire_redis_lock is called
+call_delay_in_seconds = 21600     # Expiration time for data in cache as well as redis in seconds
```

Comment:
Would it be correct to still call this as "call delay", or should we consider naming it along the lines of "expiration delay" (or "data expiration delay")?